contact_map/tests/test_plot_utils.py

```diff
@@ -6,7 +6,7 @@
 from .utils import *
 
 from contact_map.plot_utils import *
-
+from contact_map.plot_utils import _ContactPlotRange
```

Comment:
This is needed as star imports don't import private classes and variables

Reply:
Yeah, this is the reason I didn't make it a batch of suggestions. I couldn't make a suggestion on the code that hadn't changed. 😉